webview/src/experiments/components/table/TableHead.tsx

```diff
@@ -85,9 +89,14 @@ export const TableHead: React.FC<TableHeadProps> = ({
       type: MessageFromWebviewType.REORDER_COLUMNS
     })
   }
+  const [ref, scrolled] = useInView({
```

Comment:
[C] Would be good to have an indicator here to know this is for the sticky header shadow. `[ref, headerNeedsShadow]`? Just something for the next person that comes along.

Reply:
My first thought was `scrolled` because that's describing what the variable contains regardless of how we use it, but I can see how that would be helpful where it's only used in one place now.